commands/moderation/tempban.js

```diff
@@ -0,0 +1,108 @@
+const Discord = require('discord.js');
+const ms = require('ms');
+var dateFormat = require('dateformat');
+
+module.exports = {
+    name: "tempban",
+    description: "Temporarily ban a user",
+    
+    execute(msg, args, con) {
+        const user_information = /(\<@!?\d+\>)\s(\d+[yhwdms])\s(.+)$/
+
+        if (!msg.member.roles.cache.some(
+            role => role.name === "Admin" || role.name === "Moderator")) {
+                return msg.reply("You must be an Admin to use this command.");
+        }else{
+
+            if(!args.join(" ").match(user_information)) {
+                return msg.reply("The command you sent isn't in a valid format")
+            }
+
+            const [, id, timeLength, reason] = args.join(" ").match(user_information) ?? []
+
+            const toTempBan = msg.mentions.members.first();
+            if (!toTempBan) {
+                return msg.reply("Please provide a user to temporarily ban.");
+            }
+
+            if(msg.mentions.members.first() == msg.author) {
```

Comment:
I think we could use `toTempBan` here instead of `msg.mentions.members.first()`.

Reply:
Done